trade-finance-manager-api/api-tests/v1/teams/get-team-members.api-test.js

```diff
@@ -27,96 +31,108 @@ describe('GET /teams/:teamId/members', () => {
     makeRequestWithAuthHeader: (authHeader) => get(validUrlToGetTeamMembers, { headers: { Authorization: authHeader } }),
   });
 
-  it('returns a 400 response if the teamId is not one of the allowed values', async () => {
-    const unexpectedTeamId = 'unexpected team id';
+  const tfmTeams = [
+    TEAMS.BUSINESS_SUPPORT.id,
+    TEAMS.PIM.id,
+    TEAMS.RISK_MANAGERS.id,
+    TEAMS.UNDERWRITERS.id,
+    TEAMS.UNDERWRITER_MANAGERS.id,
+    TEAMS.UNDERWRITING_SUPPORT.id,
+  ];
 
-    const { status, body } = await as(tokenUser).get(`/v1/teams/${unexpectedTeamId}/members`);
+  describe.each(tfmTeams)('for teamId %s', (teamId) => {
+    it(`should returns a ${HttpStatusCode.Ok} response with only the _id, first name, and last name of active team members only returned by DTFS Central`, async () => {
+      // Arrange
+      const usersReturnedByDtfsCentral = MOCK_USERS.map(({ _id, firstName, lastName, status }, index) => ({
+        _id,
+        username: `${firstName}.${lastName}.${index}@example.com`,
+        email: `${firstName}.${lastName}.${index}@example.com`,
```

Comment:
not a blocker, could these both be a const since repeated

Reply:
Good shout, have approved.